sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/ExponentialHistogramAssert.java

```diff
@@ -47,14 +52,26 @@ public ExponentialHistogramAssert isDelta() {
     return this;
   }
 
-  /** Returns convenience API to assert against the {@code points} field. */
-  public AbstractIterableAssert<
-          ?,
-          ? extends Iterable<? extends ExponentialHistogramPointData>,
-          ExponentialHistogramPointData,
-          ?>
-      points() {
-    isNotNull();
-    return Assertions.assertThat(actual.getPoints());
+  /**
+   * Asserts the exponential histogram has points matching all of the given assertions and no more,
+   * in any order.
+   */
+  @SafeVarargs
+  @SuppressWarnings("varargs")
+  public final ExponentialHistogramAssert hasPointsSatisfying(
```

Comment:
Nit: class is already final
```suggestion
  public ExponentialHistogramAssert hasPointsSatisfying(
```

Reply:
This surprised me, but only `final` methods can have the `@SafeVarargs` annotation, without which intellij complains "Possible heap pollution from parameterized vararg type".

Shows up in the other asserts for the same reason, e.g. [HistogramAssert](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/DoubleSumAssert.java#L80), [DoubleSumAssert](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/DoubleSumAssert.java), etc.

Reply:
Huh, interesting.
Well, it doesn't really change anything anyway 🤷‍♂️